### ISSUE TITLE
The CutMix Does not work ??

### ISSUE THREAD

**Issue by liygcheng** [clovaai/CutMix-PyTorch#24]
When I employ CutMix, the results get worse instead.   What is important to keep in mind when I use CutMix?

**Comment by hellbell** [comment]
@liygcheng Could you provide more details? For example, tell me what kinds of datasets and networks you used.

**Comment by liygcheng** [comment]
dataset: CUB_200_2011 
arch: ResNet50
loss: label smooth cross entropy

**Comment by hellbell** [comment]
@liygcheng We also have tried to train models with CutMix on CUB-200 dataset, but we observed that CutMix improved the performance well. Maybe you should tune other hyper-parameters, such as learning rate, etc., to get optimal performance.
